x/farm/types/keys.go

```diff
@@ -0,0 +1,85 @@
+package types
+
+import (
+	"bytes"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/types/address"
+
+	utils "github.com/cosmosquad-labs/squad/v3/types"
+)
+
+const (
+	// ModuleName defines the module name
+	ModuleName = "farm"
+
+	// StoreKey defines the primary module store key
+	StoreKey = "f4rm" // To avoid store key collision with "farming"
+
+	// RouterKey is the message route for slashing
+	RouterKey = ModuleName
+
+	// QuerierRoute defines the module's query routing key
+	QuerierRoute = ModuleName
+)
+
+var (
+	LastPlanIdKey              = []byte{0xd0}
+	LastBlockTimeKey           = []byte{0xd1}
+	PlanKeyPrefix              = []byte{0xd2}
+	FarmKeyPrefix              = []byte{0xd3}
+	PositionKeyPrefix          = []byte{0xd4}
+	HistoricalRewardsKeyPrefix = []byte{0xd5}
+)
```

Comment:
I decided to use consecutive prefix numbers within the farm module because leaving reserved spaces within a module seems quite inefficient(unlike what I said earlier).
In future, if we need other key prefixes for the module, we can simply choose the next prefix numbers, regardless of the category.
But still, to not make key collisions between modules, we should reserve enough spaces before or after key prefixes we use in one module.

Reply:
That makes sense since we don't know how many reserved spaces we need. Let's keep it that way from now on.

Reply:
Alright!